api/v1/server/handlers/v1/workflow-runs/trigger.go

```diff
@@ -64,6 +64,10 @@ func (t *V1WorkflowRunsService) V1WorkflowRunCreate(ctx echo.Context, request ge
 				return gen.V1WorkflowRunCreate400JSONResponse(
 					apierrors.NewAPIErrors(e.Message()),
 				), nil
+			case codes.ResourceExhausted:
+				return gen.V1WorkflowRunCreate400JSONResponse(
+					apierrors.NewAPIErrors(e.Message()),
+				), nil
```

Comment:
this is the case where someone tries to trigger a workflow and is at their limit?

Reply:
yes, in this case from the REST API